tests/test_autonomy/test_images/test_runtime.py

```diff
@@ -47,6 +48,7 @@
 
 
 @skip_docker_tests
+@pytest.mark.skip(reason="Requires latest images; Unskip after the release")
```

Comment:
This test requires the autonomy image with latest open-aea, which won't happen until the release. I've tested it locally by building the images manually.